base/core/src/main/java/org/openscience/cdk/config/Isotopes.java

```diff
@@ -85,4 +94,56 @@ private Isotopes() throws IOException {
             add(isotope);
         }
     }
+
+    private static boolean isMajor(IIsotope atom) {
```

Comment:
Can this method be made public? I think it can be useful outside this class too... what do you think?

Reply:
I think okay as private for now, can always make it public in future... primarily waiting on https://github.com/cdk/cdk/issues/458